src/app/components/ModeModal.ts

```diff
@@ -2,10 +2,10 @@ import { BasicModal } from './BasicModal';
 import { Render } from '../utils/Render';
 import { PlayerDTO } from '~src/Interfaces/PlayerDTOInterface';
 import { CallbackOneParam } from '~src/Interfaces/CallbackOneParamInterface';
-
+import { Avatars } from '~src/Enums/AvatarsEnum';
```

Comment:
Check if you should remove tildas from paths

Reply:
It's from autoimport. Is it something wrong with it?